components/LoginForm/LoginForm.js

```diff
@@ -49,6 +51,9 @@ function LoginForm({ initialValues, login, onSuccess }) {
       await onSuccess({ token });
       actions.setSubmitting(false);
       actions.resetForm();
+      if (redirectFunc) {
+        redirectFunc();
+      }
```

Comment:
If you set the default to `lodash/noop` you can simply always call `redirectFunc`

Reply:
✔️